units/en/unit1/rl-framework.mdx

```diff
@@ -83,12 +81,10 @@ The actions can come from a *discrete* or *continuous space*:
 
 <figure>
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/mario.jpg" alt="Mario">
-<figcaption>Again, in Super Mario Bros, we have only 5 possible actions: 4 directions and jumping</figcaption>
+<figcaption>In Super Mario Bros, we have only 4 possible actions: left, right, jumping and ducking</figcaption>

```

Comment:
Can you change ducking to jumping? It's easier to understand for most readers

Reply:
By ducking I meant crounching, opposite direction of jumping. Should I just change it to left, right, up and down? "Up" would be jumping and "down" would be ducking/crouching. I could also place between brackets the action associated with the direction. Being that said, the options are:

**In Super Mario, we only have 4 possible actions: ...**

1. left, right, up and down.
2. left, right, up (jumping) and down (crouching).


If I got something wrong, leave a third suggestion and I'll go with that :)

Reply:
I like this one : left, right, up (jumping) and down (crouching). Thanks 🤗